### ISSUE TITLE
Cannot Save Files "python2.exe: There is no disk in the drive. Please insert a disc into drive E:"

### ISSUE THREAD

**Issue by MeTheMightyMe** [mypaint/mypaint#950]
Hello. I have just started using MyPaint today and I am trying to save my first project. However, I receive an error when I try to Save As or Save. When I go to save my project, a window pops up saying:

"python2w.exe - No Disk

There is no disk in the drive. Please insert a disk into drive E:"

No matter what button I click (Cancel, Try Again, Continue, or X), it brings me to the MyPaint folder location.

I have searched all over the place and I cannot seem to find an article or answer that is specific to my current issue.

#### Basic system details

MyPaint version:: The latest version available, just downloaded it today. 
Operating system: Windows 10 Home 64-bit
Desktop environment: 
Intel Core i7-7700 CPU @ 4.20GHz processor, 
NVIDA GeForce GTX 1080 video driver, 
Acer ET430K-4 (NVIDIA High Definition Audio), 
System Manufacturer: PowerSpec

#### Steps to reproduce

1. Start MyPaint
2. Click Save As or Save
3. Error box pops up, I click any button 5 times.
4. A window pops up that only lets me select pre-existing files. (Picture attached).

#### Backtraces or error messages

```
python2.exe - No Disk

There is no disk in the drive. Please insert a disk into drive E:.
```

**Comment by MeTheMightyMe** [comment]
Ha, you are right! Haha, I totally overlooked the box at the top. The layout threw me off completely. Pardon for me for overlooking this! 

I have successfully saved my project. I am glad it didn't prevent me from doing so after all!

I tried your suggestion of downloading the alpha executable for Windows 64-bit. 

The python2.exe error still persists in this alpha version as well. 

I have edited the topic appropriately to reflect this persistant error/bug. It is still a minor nuisance that can be worked around, but it would be nice to resolve it to keep things running smoothly.

Thank you for your time.
